static/src/javascripts-legacy/projects/commercial/modules/creatives/fabric-v1.js

```diff
@@ -97,6 +97,10 @@ define([
                 this.scrollingBg.style.backgroundAttachment = 'fixed';
             }
 
+            // #? `classList.add` takes multiple arguments, but we are using it
+            // here with arity 1 because polyfill.io has incorrect support with IE 10 and 11.
+            // One may revert to this.adSlot.classList.add('ad-slot--fabric-v1', 'ad-slot--fabric', 'content__mobile-full-width');
+            // We support is correct or se stop supporting IE <= 11
```

Comment:
Corrected. Should have been "When support is correct or when we stop supporting IE <= 11"